starcounter-include.html

```diff
@@ -203,24 +203,29 @@
          * @param  {String} path Polymer notification path
          * @param  {Mixed} value New value
          */
-        LauncherPartialPrototype._notifyPath = function(path, value) {
+        StarcounterIncludePrototype._notifyPath = function(path, value) {
```

Comment:
I wrote that in regexp just for clarity. Same can be achieved with an `if`:

```js
if (path.indexOf("partial.") === 0) {
  path = path.replace("partial.", "model.");
}
else if (path.indexOf("viewModel.") === 0) {
  path = path.replace("viewModel.", "model.");
} 
```

The performance of string replace is close to 0. Polymer does more expensive things in every notification anyway: https://github.com/Polymer/polymer/blob/48fdac7f795c17df154fd7f64830cd6182426536/lib/mixins/property-effects.html#L251